html/docs/files.html

```diff
@@ -95,6 +95,19 @@ <h4>Setup</h4>
               only affects your ability to <code>undo</code> changes.
             </p>
 
+						<p> 
+							The <code>tags.data</code> contains information about how often a
+							certain tag is used to make Timewarrior able to notify the user
+							whether the tag is new or not. In contrast to
+							<code>undo.data</code> you should not modify the file directly to
+							avoid inconsistencies in the database. When <code>tags.data</code>
+							gets out of sync with the interval data you can delete it and
+							Timewarrior will recreate it from the interval data.  More
+							information at <a
```

Comment:
Github Issues should not be referenced in documentation. Important information from Github discussions should rather be consolidated here.